src/HomeWork/hw8/Mail/MailService.java

```diff
@@ -0,0 +1,32 @@
+package HomeWork.hw8.Mail;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class MailService<T> implements Consumer<Sendable<T>> {
+
+    private Map<String, List<T>> mailBox;
+
+    public MailService() {
+        mailBox = new HashMap<>() {
+            @Override
+            public List<T> get(Object key) {
+                return super.get(key) == null ? new ArrayList<>() : super.get(key);
```

Comment:
better move this logic where it is use without overriding HashMap

Reply:
i changed this, but now i have 1 failed assert cause NPE (
class Test => method MessageStreamTest() => commented line
mailBox.get(randomTo) = null and i dont know how to solve it correctly